desktop/apps/auction2/components/filter_sort/index.js

```diff
@@ -0,0 +1,54 @@
+import { default as React, PropTypes } from 'react';
+import { connect } from 'react-redux'
+import _ from 'underscore'
+import { updateSort } from '../../actions'
+
+function FilterSort({ sortMap, filterParams, updateSortAction }) {
+  const selectedSort = filterParams.sort
+  return (
+    <div className='auction2-filter-sort'>
+      <span className='auction2-filter-sort__label bordered-pulldown-label'>Sort by:</span>
+      <div className='bordered-pulldown'>
+        <a className='bordered-pulldown-toggle' href='#'>
+          <span className='bordered-pulldown-text'>{ sortMap[selectedSort] }</span>
+          <div className='bordered-pulldown-toggle-caret'>
+            <span className='caret'></span>
+          </div>
+        </a>
+        <div className='bordered-pulldown-options'>
+          {
+            _.map(sortMap, (sortName, sort) => {
```

Comment:
`map` is native to JS -- should always defer to native methods and then to underscore. Most simple operations in JS these days can be achieved without underscore.

```javascript
{
  sortMap.map((sortName, sort) => { ... }
}
```